student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, parent_permission, name, classroom)
```

Comment:
- [ ]  The name' and 'parent_permission'  should have default values of "Unknown" and 'true', please add it
```suggestion
  def initialize(`age, classroom, name = "Unknown", parent_permission = true`)
```

Reply:
Hi @prolajumokeoni . Thanks for your suggestion.